R/helpers-vector.R

```diff
@@ -88,6 +88,12 @@ all_of <- function(x) {
 #' @export
 any_of <- function(x, ..., vars = NULL) {
   vars <- vars %||% peek_vars(fn = "any_of")
-  check_dots_empty()
+  if (!missing(...)) {
+    abort(c(
```

Comment:
Use `cli_abort()` with `.code` style?

Reply:
How about I convert them all in a separate PR?

Reply:
I see you filed #285 so we're thinking the same way 😄

Reply:
Up to you. Personally for this sort of things I generally convert any code I touch ahead of time, before the bulk changes. Especially for this sort of dull but tricky changes that are so easy to mess up when doing a mass update.

Reply:
Yeah, good point.